specificatie/zoek-personen.yaml

```diff
@@ -0,0 +1,333 @@
+              type: string
+              pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{1,200}(\?+|\*)?$
+              example: "Vries"
+            geslachtsaanduiding:
+              description: |
+                Geeft aan dat de persoon een man of een vrouw is, of dat het geslacht (nog) onbekend is.
+              type: string
+              pattern: ^([Mm]|[Vv]|[Oo])$
+              example: "V"
+            voorvoegsel:
+              description: |
+                Deel van de geslachtsnaam dat vooraf gaat aan de rest van de geslachtsnaam. Het zoeken op het voorvoegsel is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
+              type: string
+              pattern: ^[a-zA-Z0-9À-ÿ' '\-\']{1,10}$
+              example: "de"
+            voornamen:
+              description: |
+                De verzameling namen die, gescheiden door spaties, aan de geslachtsnaam voorafgaat. ** Bij deze query-parameter is het gebruik van een [wildcard](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature) toegestaan in combinatie met minimaal 2 karakters.** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
+              type: string
+              pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ' '\-\']{1,200}(\?+|\*)?$
+              example: "Dirk"
+    ZoekMetNaamEnGemeenteVanInschrijving:
+      required:
+        - geslachtsnaam
+        - gemeenteVanInschrijving
+        - voornamen
+      allOf:
+        - $ref: '#/components/schemas/PersonenQuery'
+        - type: object
+          properties:
+            inclusiefOverledenPersonen:
+              $ref: '#/components/schemas/InclusiefOverledenPersonen'
+            geslachtsaanduiding:
+              description: |
+                Geeft aan dat de persoon een man of een vrouw is, of dat het geslacht (nog) onbekend is.
+              type: string
+              pattern: ^([Mm]|[Vv]|[Oo])$
+              example: "V"
+            geslachtsnaam:
+              description: |
+                De (geslachts)naam waarvan de eventueel aanwezige voorvoegsels zijn afgesplitst. **Gebruik van de wildcard is toegestaan. Zie [feature-beschrijving](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
+              type: string
+              pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{1,200}(\?+|\*)?$
```

Comment:
wat hadden we hierover ook weer afgesproken? Ik dacht dat je niet mag beginnen met een wildcard, en dat bij gebruik van een wildcard ten minste 3 tekens moeten worden opgegeven: ^([a-zA-Z0-9À-ÿ '\-\']{1,200}|([a-zA-Z0-9À-ÿ '\-\']{3,199}(\?+|\*)?))$

Reply:
aangepast. Toegevoegd dat max 5 "?" wildcard tekens kan worden opgegeven